geode-core/src/main/java/org/apache/geode/management/internal/ManagementAgent.java

```diff
@@ -396,12 +402,12 @@ public JMXServiceURL getAddress() {
 
           @Override
           public synchronized void start() throws IOException {
+            super.start();
```

Comment:
Why did this method change? It used to bind first and then called super.start. Now it calls start first and then does a bind which does not seem correct.